cmd/mistry/Gopkg.lock

```diff
@@ -1,12 +1,6 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```

Comment:
Use BuildResult (as opposed to buildResult) and BuildInfo in the commit message.

Reply:
Done